src/containers/b2b/RequisitionPageMain.tsx

```diff
@@ -210,74 +490,89 @@ class RequisitionPageMain extends Component<RequisitionPageMainProps, Requisitio
                       {' '}
                       {intl.get('item-selected')}
                     </p>
                   </div>
-                  <button type="button" className="ep-btn small delete-btn">{intl.get('delete')}</button>
+                  <button type="button" className="ep-btn small delete-btn" onClick={this.handleBulkDelete}>{intl.get('delete')}</button>
                   {this.renderDropdownMenu()}
                 </div>
               )}
               <div className="product-pagination">
-                <button type="button" className="pagination-btn prev-btn" disabled>
+                {() => {
+                  if (isPageLoading) return (<div className="miniLoader" />);
+                  return '';
+                }}
+                <button type="button" className="pagination-btn prev-btn" onClick={() => { this.handlePagination(productsData._previous); }} disabled={!(productsData && productsData._previous)}>
                   <ArrowLeft className="arrow-left-icon" />
                 </button>
                 <span className="pagination-txt">
-                  Page 1 of 3
+                  {intl.get('pagination-message', { current: pagination.current, pages: pagination.pages })}
                 </span>
-                <button type="button" className="pagination-btn next-btn">
+                <button type="button" className="pagination-btn next-btn" onClick={() => { this.handlePagination(productsData._next); }} disabled={!(productsData && productsData._next)}>
                   <ArrowLeft className="arrow-left-icon" />
                 </button>
               </div>
             </div>
-            <div className={`product-table ${multiSelectMode ? 'multi-select-mode' : ''}`}>
-              <div className="product-table-heading">
-                <div className="product-table-heading-item" />
-                <div className="product-table-heading-item">
-                  <span>{intl.get('product')}</span>
-                </div>
-                <div className="product-table-heading-item" />
-                <div className="product-table-heading-item">
-                  <span>{intl.get('quick-order-sku-title')}</span>
-                </div>
-                <div className="product-table-heading-item">
-                  <span>{intl.get('quantity')}</span>
-                </div>
-                <div className="product-table-heading-item">
-                  <span>{intl.get('product-options')}</span>
-                </div>
-                <div className="product-table-heading-item">
-                  <span>{intl.get('price')}</span>
-                </div>
-                <div className="product-table-heading-item actions">
-                  <span>{intl.get('actions')}</span>
+            {isPageLoading ? (
+              <div className="loader" />
+            ) : (
+              <div className={`product-table ${multiSelectMode ? 'multi-select-mode' : ''}`}>
```

Comment:
We'll need to consider how we display this table for mobile. Currently the titles are left-justified
![image](https://user-images.githubusercontent.com/39800563/73613526-a86f6280-45c4-11ea-9d8d-448fda6291b6.png)